### ISSUE TITLE
No activity since 2019

### ISSUE THREAD

**Issue by renatoagds** [Dreamscapes/eslint-import-resolver-lerna#10]
That's some plan to return the activity of this repo @robertrossmann ?

We have a few pull requests that still open for a long time.

**Comment by chrisjlebron** [comment]
Just piping in here cuz I've been looking into this myself: looks like at least for the monorepo use case, this is now built in by specifying `packageDir`:
https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-extraneous-dependencies.md

So for example, in my lerna monorepo with the structure
```
my-monorepo/
  package.json
  .eslintrc.js
  packages/
    my-package/
      package.json
      .eslintrc.js
```
in the nested `eslintrc.js` I added this rule

```
'import/no-extraneous-dependencies': [
  'error',
  { packageDir: ['./', '../../'] }
]
```
telling eslint-plugin-import to look for my package.json at the package level and at the monorepo root level.

Hope that helps!

**Comment by robertrossmann** [comment]
Weeey hey this project is back from the graveyard! 🎉 I'm guessing COVID-19 lockdowns do have some benefits after all (got a free weekend).

I just published v2 which is a complete TypeScript rewrite plus all of the reported bugs have been fixed. Plus I realised there are more bugs people have not discovered yet so I fixed those, too. I added a test suite to make sure! ✅

I hope you enjoy this awesome new release and let me know if stuff doesn't work as expected! I promise I won't go AWOL for another 2 years!
